esphome/components/veml7700/sensor.py

```diff
@@ -0,0 +1,177 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import i2c, sensor
+from esphome.const import (
+    CONF_ID,
+    CONF_GAIN,
+    CONF_GLASS_ATTENUATION_FACTOR,
+    CONF_ACTUAL_GAIN,
+    CONF_AUTO_MODE,
+    CONF_FULL_SPECTRUM,
+    CONF_INFRARED,
+    CONF_INTEGRATION_TIME,
+    UNIT_LUX,
+    UNIT_MILLISECOND,
+    ICON_BRIGHTNESS_5,
+    ICON_BRIGHTNESS_6,
+    ICON_TIMER,
+    DEVICE_CLASS_ILLUMINANCE,
+    STATE_CLASS_MEASUREMENT,
+)
+
+CODEOWNERS = ["@latonita"]
+DEPENDENCIES = ["i2c"]
+
+UNIT_COUNTS = "#"
+ICON_GAIN = "mdi:multiplication"
```

Comment:
Please name consts with what actually represents the value
```suggestion
ICON_MULTIPLICATION = "mdi:multiplication"
```

Reply:
Marked as resolved, but it seems this change was not made?

Reply:
Indeed

Reply:
apologies, merge glitch. fixed now.